mesonbuild/msubprojects.py

```diff
@@ -174,11 +174,17 @@ def foreach(wrap, repo_dir, options):
         mlog.log('  -> Not downloaded yet')
         return
     try:
-        subprocess.check_call([options.command] + options.args, cwd=repo_dir)
-        mlog.log('')
+        out = subprocess.check_output([options.command] + options.args,
```

Comment:
```python
out = subprocess.check_output([options.command] + options.args, 
                                      stderr=subprocess.STDOUT,
                                      universal_newlines=True,
                                      cwd=repo_dir)
```

Reply:
@scivision thanks for the comment, I get that `universal_newlines=True` makes `.decode()` unnecessary, but is there also some other benefit? I am trying to be consistent with what the `git()` function is doing.

Maybe the change you propose can be a followup where all instances of `check_output().decode()` are replaced.

Reply:
OK